pkgs/development/libraries/libpulsar/default.nix

```diff
@@ -0,0 +1,98 @@
+{ boost17x
+, clang
+, clang-tools
+, cmake
+, curl
+, enablePython ? false
+, fetchurl
+, gcc
```

Comment:
Does that make sense on darwin?

Also I guess it's part of stdenv, so doesn't need to be listed explicitly.

Reply:
I have removed both `gcc`, and `clang` from the argument list. You are right, they don't need to be listed explicitly.